src/main.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(lazy_cell)]
```

Comment:
Here's the inclusion on `main.rs` of the unstable feature, in case we want to do this. Let's continue the conversation on the pull request itself!